README.md

```diff
@@ -45,14 +45,18 @@ The player then attempts to catch them hammer after it flips at least once in th
 The number of vertical flips is the number of strikes the player may make in the Strike phase.
 
 
-The player may not adjust their grip on the hammer. 
+The player may not adjust their grip on the hammer.
 The way that player catches the hammer is how they must perform their strikes.
+If they adjust the grip on their hammer they must drink and immediately end their turn.
+If they attempt to perform a strike (hammer makes contact with the stump or a nail) after regripping they also lose their next turn.
 
 
 Upon catching the hammer, the player must immediately move to the Strike phase without hesitation.
 
 
-If a player fails to catch the hammer, they must drink and immediately end their turn, passing the hammer to the next play in turn order.
+If a player fails to catch the hammer they must drink and immediately end their turn, passing the hammer to the next play in turn order.
```

Comment:
Yeah you right. I always second guess my self about the if, then stuff. I'll fix it